### DISCUSSION TITLE
Ability to disable twilio signature verification via configuration setting

### DISCUSSION THREAD

**Discussion by tonyschwartz** [target/goalert#2618] [General]
Enhancement request:  Please allow us to disable the Twilio signature verification.  We receive the twilio callback in a secure application in our DMZ which does the signature verification itself.  Then, a completely separate app needs to pick up that message and deliver it to the GoAlert system.  I am protecting GoAlert completely from any external threats.  I wish to remove/disable verification.  If that is unacceptable, I need some way to alter the host because it's completely different from what Twilio sees.  Thank you for your consideration.

**Comment by mastercactapus** [comment]
I can see the use case; we've run into this a couple of times as well.

I feel like it may be better as a CLI flag/env variable though. That way disabling the security check would happen based on the environment rather than the global application config.
